rpcs3/util/types.hpp

```diff
@@ -1056,6 +1056,41 @@ template <typename T, usz Size>
 	return static_cast<u32>(Size);
 }
 
+template <typename T, usz Size, typename U>
+[[nodiscard]] constexpr auto& at32(const T (&array)[Size], u32 index,
```

Comment:
Maybe use std::size.

Reply:
I mean this overload is better removed in this widely used header, and std::size can help with that. There's concept test in serializtion.hpp for detection of arrays, even worjs for std::span etc.

Reply:
Okay, I'll look into it